<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ImageMorphology</class>
    <widget class="QWidget" name="ImageMorphology">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>图像形态学</string>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="2">
                <widget class="QTabWidget" name="tabWidget">
                    <property name="currentIndex">
                        <number>2</number>
                    </property>
                    <widget class="QWidget" name="binarizationTab">
                        <attribute name="title">
                            <string>二值化</string>
                        </attribute>
                        <layout class="QGridLayout" name="gridLayout_2">
                            <item row="0" column="0">
                                <widget class="QLabel" name="binarizedImageLabel">
                                    <property name="text">
                                        <string>请点击下方按钮打开图像</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="filterTab">
                        <attribute name="title">
                            <string>滤波</string>
                        </attribute>
                        <layout class="QGridLayout" name="gridLayout_4">
                            <item row="0" column="0">
                                <widget class="QLabel" name="filteredImageLabel">
                                    <property name="text">
                                        <string>请点击下方按钮打开图像</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="skeletonizationTab">
                        <attribute name="title">
                            <string>细化</string>
                        </attribute>
                        <layout class="QGridLayout" name="gridLayout_3">
                            <item row="0" column="0">
                                <widget class="QLabel" name="skeletonizedImageLabel">
                                    <property name="text">
                                        <string>请点击下方按钮打开图像</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
            <item row="1" column="0" colspan="2">
                <widget class="QPushButton" name="openImageButton">
                    <property name="text">
                        <string>打开图像</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
